src/App.jsx

```diff
@@ -38,7 +38,7 @@ export function App() {
 	 * This custom hook takes our token and fetches the data for our list.
 	 * Check ./api/firestore.js for its implementation.
 	 */
-	const data = useShoppingListData(listPath);
+	const data = useShoppingListData('/A45I0SLfsWeHLDg47ETZM5vP8fG2/test-list');
```

Comment:
Once `SingleList` receives all the props, we should be able to remove the hardcoded list path and replace it with `listPath` again.